src/wallet.js

```diff
@@ -869,17 +870,21 @@ const wallet = {
    * @inner
    */
   markWalletAsStarted() {
-    return storage.setItem('wallet:started', true);
+    return StorageProxy.getStorage().setItem('wallet:started', true);
   },
 
   /**
    * Subscribe to receive updates from an address in the websocket
    *
    * @param {string} address
    */
-  subscribeAddress(address) {
+  subscribeAddress(address, connection = null) {
     const msg = JSON.stringify({'type': 'subscribe_address', 'address': address});
-    WebSocketHandler.sendMessage(msg);
+    WebSocketHandler.ws.sendMessage(msg);
+
+    if (connection) {
+      connection.websocket.sendMessage(msg);
```

Comment:
Why are we subscribing to two connections?

Reply:
The old wallet code still uses the `WebSocketHandler`, which has a websocket connection. Our new code has a connection instance, which has a different websocket connection. To receive the message in both connections I should subscribe both.

Reply:
I got it. But it won't be needed if we remove `WebSocketHandler`. I think you should either drop a comment about it or remove the `WebSocketHandler` from the lib and update the wallets.

Reply:
I don't think we should think about removing the `WebSocketHandler` for now. It's used in the whole library and wallets. It will be removed soon in the future. For now I left a comment in the code, as you suggested.